framework/src/modules/chain/components/storage/entities/round_delegates.js

```diff
@@ -120,7 +128,7 @@ class RoundDelegates extends BaseEntity {
 	 * @param {Object} [tx]
 	 * @returns {Promise.<boolean, Error>}
 	 */
-	delete(filters, _options, tx = null) {
+	delete(filters, tx = null) {
```

Comment:
Why we remove the options parameter? Maybe it makes sense to follow the consistent interface for the entities.

Reply:
But it's not used anywhere in the method? :D